lib/gems/pending/metadata/util/win32/system_path_win.rb

```diff
@@ -95,7 +95,7 @@ def self.parse_bcd(fs)
       # Decode disk signature
       unless result[:os_device].nil?
         d = result[:os_device].split(',')
-        result[:disk_sig] = (d[59] + d[58] + d[57] + d[56]).to_i(16)
+        result[:disk_sig] = (d[59] + d[58] + d[57] + d[56]).to_i(16) if d.size > 60
```

Comment:
I can send another PR for this change. It's a valid disk even d.size <= 60. In our case here, d.size = 22. Except this signature, all other information are meaningful and valid. 

Actually I didn't find this os_device signature is used anywhere later.